spec/spec_helper.rb

```diff
@@ -59,28 +59,18 @@ def header_sanitizer(response_header, transient_headers)
 end
 
 VCR.configure do |config|
-  # config.debug_logger = $stdout # Keep for debugging tests.
-
-  # Configure VCR to use rspec metadata.
-  config.hook_into(:webmock)
-  config.configure_rspec_metadata!
-
   config.ignore_hosts('codeclimate.com') if ENV['CI']
   config.cassette_library_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.root, 'spec/vcr_cassettes')
+  config.hook_into(:webmock)
```

Comment:
This is defined by core, so it's possible we don't need this.

Reply:
Yeah we can probably drop it, can merge as-is since it isn't hurting anything